Client/Frontend/Browser/TabToolbar.swift

```diff
@@ -235,19 +238,17 @@ class TabToolbar: Toolbar, TabToolbarProtocol {
         forwardButton.accessibilityIdentifier = "TabToolbar.forwardButton"
         stopReloadButton = ToolbarButton()
         stopReloadButton.accessibilityIdentifier = "TabToolbar.stopReloadButton"
-        shareButton = ToolbarButton()
-        shareButton.accessibilityIdentifier = "TabToolbar.shareButton"
-        bookmarkButton = ToolbarButton()
-        bookmarkButton.accessibilityIdentifier = "TabToolbar.bookmarkButton"
+        tabsButton = TabsButton()
+        tabsButton.accessibilityIdentifier = "TabToolbar.tabsButton"
         menuButton = ToolbarButton()
         menuButton.accessibilityIdentifier = "TabToolbar.menuButton"
-        actionButtons = [backButton, forwardButton, stopReloadButton, shareButton, menuButton]
+        actionButtons = [backButton, forwardButton, stopReloadButton, tabsButton, menuButton]
 
         super.init(frame: frame)
 
         self.helper = TabToolbarHelper(toolbar: self)
 
-        addButtons(backButton, forwardButton, stopReloadButton, shareButton, menuButton)
+        addButtons(backButton, forwardButton, stopReloadButton, tabsButton, menuButton)
```

Comment:
nit: I think this func should take an array (looks like varargs currently) and then re-use the button array from line 245